dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/TaskConstants.java

```diff
@@ -398,4 +403,19 @@ private TaskConstants() {
     public static final String AWS_SECRET_ACCESS_KEY= "aws.secret.access.key";
     public static final String AWS_REGION = "aws.region";
 
+    /**
+     * use for k8s task
+     */
+    public static final String API_VERSION = "batch/v1";
+    public static final String IMAGE_PULL_POLICY = "Always";
+    public static final String RESTART_POLICY = "Never";
```

Comment:
Is it better to be configurable?

Reply:
1. The api version is defined by the k8s dependency in task module
2. set the value of policy to Always to get latest image for each running
3. ds can retry task when the task failed, so there is no need to take k8s solution

Reply:
1. The api version is defined by the k8s dependency in task module
2. set the value of policy to Always to get latest image for each running
3. ds can retry task when the task failed, so there is no need to take k8s solution